app/views/CinematicCanvas.vue

```diff
@@ -28,6 +28,10 @@ export default {
     enterDisabled: true
   }),
   mounted: function() {
+    if (!me.isAdmin()) {
```

Comment:
Alternatively, we can create a feature flag for this and do the admin check there like this: https://github.com/codecombat/codecombat/blob/master/app/models/User.coffee#L584

Reply:
That's a brilliant idea.